hail/src/main/scala/is/hail/io/BufferSpecs.scala

```diff
@@ -151,6 +153,23 @@ final case class LZ4FastBlockBufferSpec(blockSize: Int, child: BlockBufferSpec)
   def typeName = "LZ4FastBlockBufferSpec"
 }
 
+final case class LZ4SizeBasedBlockBufferSpec(hc: Boolean, blockSize: Int, minCompressionSize: Int, child: BlockBufferSpec)
```

Comment:
This is fine, but something like `compressorType: String` seems more forward extensible.